htdocs/core/modules/modProduct.class.php

```diff
@@ -194,7 +194,7 @@ public function __construct($db)
 		);
 		if (is_object($mysoc) && $mysoc->useNPR()) $this->export_fields_array[$r]['p.recuperableonly'] = 'NPR';
 		if (!empty($conf->fournisseur->enabled) || !empty($conf->margin->enabled)) $this->export_fields_array[$r] = array_merge($this->export_fields_array[$r], array('p.cost_price'=>'CostPrice'));
-		if (!empty($conf->stock->enabled)) $this->export_fields_array[$r] = array_merge($this->export_fields_array[$r], array('p.stock'=>'Stock', 'p.seuil_stock_alerte'=>'StockLimit', 'p.desiredstock'=>'DesiredStock', 'p.pmp'=>'PMPValue'));
+		if (!empty($conf->stock->enabled)) $this->export_fields_array[$r] = array_merge($this->export_fields_array[$r], array('e.ref'=>'DefaultWarehouse', 'p.tobatch'=>'ManageLotSerial', 'p.stock'=>'Stock', 'p.seuil_stock_alerte'=>'StockLimit', 'p.desiredstock'=>'DesiredStock', 'p.pmp'=>'PMPValue'));
```

Comment:
Adding the warehouse here will make the profile export to not being the profile to export product but to export stock into warehouse (you will get n lines for each warehouse).
There is another dedicated profile for that.

Reply:
What ?
e.ref is warehouse name from llx_entrepot table simply joined by warehouse ID, not the stock !
You already merged my PR on develop back in August => I didn't know develop was for v13 back then.
You make me doubt. I tested it on develop branch and it works fine:
![image](https://user-images.githubusercontent.com/69054301/100921207-82fcff00-34dc-11eb-8986-8d130f74acca.png)